sentry-samples/sentry-samples-android/src/early/java/io/sentry/samples/android/MyApplication.java

```diff
@@ -0,0 +1,53 @@
+package io.sentry.samples.android;
```

Comment:
I'd rather roll back the sample changes, and focus only on the classes visibility.